proposals/0064-transaction-receipt.md

```diff
@@ -0,0 +1,257 @@
+---
+simd: '0064'
+title: Transaction Receipts
+authors:
+  - Anoushk Kharangate (Tinydancer)
+  - Harsh Patel (Tinydancer)
+  - Richard Patel (Jump Firedancer)
+category: Standard
+type: Core
+status: Draft
+created: 2023-06-20
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This proposal introduces two new concepts to the Solana runtime.
+
+- Receipts, a deterministic encoding of state changes induced by a transaction;
+- The receipt tree, a commitment scheme over all transaction receipts in a slot.
```

Comment:
can we explicitly use "transaction receipt" instead of "receipt" shorthand throughout the document? we've backed ourselves into terminology corners enough times with laziness that it's best to drive home an unambiguous name early